### ISSUE TITLE
How to re-sync folders synced with MEGAsync through MEGAcmd?

### ISSUE THREAD

**Issue by DanyGee** [meganz/MEGAcmd#104]
Scenario:
Elementary OS drops system tray completely in JUNO.
That means MEGAsync will have problems working properly.
It works, but you can't hide the window, you can only close the client. I disabled its autostart for now.
I already have my MEGA folder synced through MEGAsync - /home/dany/MEGA.
Now I would like to resync it with MEGAcmd.
What would be the easiest way to achieve that?
Already installed MEGAcmd :heavy_check_mark: 
Already logged in :heavy_check_mark: 
Made MEGAcmd server start in BG after login with *desktop file in autostart :heavy_check_mark: 
I know I should now sync with _**sync [localpath dstremotepath]**_ but don't know how the root folder on remotepath should look like. Is it **_root/_** or something else:question: 
Or maybe there's more I'm missing:question: 
In other words, need to find a way to sync existing MEGA folder without systray and in the background:bulb:

**Comment by DanyGee** [comment]
OK, found it myself :1st_place_medal: 

The proper command was:
`mega-sync /home/dany/MEGA /`

Sync status looks:ok_hand: 
```
mega-sync
ID LOCALPATH      REMOTEPATH     ActState   SyncState     SIZE  FILES   DIRS
 0 /home...y/MEGA /              Active     Synced     4.21 GB  11241    689
```
